javascript/javascript3/week3/screenshot.js

```diff
@@ -0,0 +1,103 @@
+console.log("Screenshot API");
+
+// STEP 1: FETCH DATA FROM API
+//Fetch Data from API
+
+// Select the button, when the button is clicked, get the input value.
+const searchButton = document.querySelector("#generate-screenshot");
+
+// Display Screenshot
+const displayScreenshot = document.querySelector("#image");
+
+// Save Screenshot
+const saveScreenshotButton = document.querySelector(".saveScreenshot");
+
+const getScreenshot = async (event) => {
+    // Search Input
+    const searchInput = document.querySelector("#url").value
+
+    if (!searchInput) {
```

Comment:
Could you try to ensure they have entered a valid URL using a [regular expression](https://stackoverflow.com/questions/3809401/what-is-a-good-regular-expression-to-match-a-url)?